packages/keystatic/src/form/fields/markdoc/editor/editor-footer.tsx

```diff
@@ -0,0 +1,218 @@
+import { useCallback, useMemo } from 'react';
+
+import { ActionButton } from '@keystar/ui/button';
+import { Dialog, DialogTrigger } from '@keystar/ui/dialog';
+import { Icon } from '@keystar/ui/icon';
+import { imageIcon } from '@keystar/ui/icon/icons/imageIcon';
+import { typeIcon } from '@keystar/ui/icon/icons/typeIcon';
+import { HStack } from '@keystar/ui/layout';
+import { Content, SlotProvider } from '@keystar/ui/slots';
+import { Heading, Kbd, Prose, Text } from '@keystar/ui/typography';
+
+import { getUploadedFileObject } from '../../image/ui';
+import { useEditorDispatchCommand } from './editor-view';
+import { readFileAsDataUrl } from './images';
+
+export function EditorFooter() {
+  const runCommand = useEditorDispatchCommand();
+  const handleImagePress = useCallback(async () => {
+    const file = await getUploadedFileObject('image/*');
+    if (!file) return;
+    const src = await readFileAsDataUrl(file);
+    runCommand((state, dispatch) => {
+      if (dispatch) {
+        dispatch(
+          state.tr.replaceSelectionWith(
+            state.schema.nodes.image.createChecked({
+              src,
+              filename: file.name,
+            })
+          )
+        );
+      }
+      return true;
+    });
+  }, [runCommand]);
+
+  const slots = useMemo(
+    () => ({ icon: { size: 'small' }, text: { size: 'small' } }) as const,
+    []
+  );
+
+  return (
+    <HStack gap="small" margin="small">
+      <DialogTrigger isDismissable>
+        <ActionButton prominence="low">
+          <SlotProvider slots={slots}>
+            <Icon src={typeIcon} />
+            <Text>Markdown support</Text>
+          </SlotProvider>
+        </ActionButton>
+        <MarkdownDialog />
+      </DialogTrigger>
+      <ActionButton prominence="low" onPress={handleImagePress}>
+        <SlotProvider slots={slots}>
+          <Icon src={imageIcon} />
+          <Text>Paste, drop, or click to add images</Text>
+        </SlotProvider>
+      </ActionButton>
+    </HStack>
+  );
+}
+
+function MarkdownDialog() {
+  return (
+    <Dialog size="large">
+      <Heading>Basic writing and formatting syntax</Heading>
+      <Content>
+        <Prose size="regular">
+          <p>
+            Markdown is a way to style text; it is intended to be as
+            easy-to-read and easy-to-write as is feasible.
+          </p>
```

Comment:
Feels weird to say this when you're not reading markdown, there are just some markdown shortcuts